davisinteractive/connector/abstract.py

```diff
@@ -10,7 +10,7 @@ def get_scribble(self, sequence, scribble_idx):
         raise NotImplementedError('This is an abstract class')
 
     def post_predicted_masks(self, sequence, scribble_idx, pred_masks, timming,
-                             interaction):
+                             interaction, next_scrrible_frame_candidates=None):
```

Comment:
It should be next_scribble_frame_candidates

Reply:
Done